app/views/admin/invoices/show.html.erb

```diff
@@ -1 +1,7 @@
-<h3> Admin Invoices Show Page (placeholder)</h3>
\ No newline at end of file
+<section id="invoice-details">
+  <h2><%= "Invoice ##{@invoice.id}"%></h2>
+  <p><%= "Status: #{@invoice.status}"%></p>
+  <p><%= "Created on: #{@invoice.created_at.strftime("%A, %B %d, %Y")}"%></p>
+  <h4>Customer:</h4>
```

Comment:
Refactor idea for us - Make a helper method called "full_name" that combines the two names